python/lsst/ts/atspectrograph/model.py

```diff
@@ -114,8 +114,8 @@ def __init__(self, log):
         self.cmd_lock = asyncio.Lock()
         self.controller_ready = False
 
-        self.filters = dict()
-        self.gratings = dict()
+        self.filter_to_enum_mapping_dict = dict()
+        self.gratings_to_enum_mapping_dict = dict()
```

Comment:
As I said in the csc, having the type in the variable name is considered and anti-pattern in python. I would recommend renaming these two to:

```
self.filter_to_enum
self.grating_to_enum
```

Reply:
i renamed to `X_to_enum_mapping`